controllers/managedmcg_controller.go

```diff
@@ -72,61 +69,68 @@ type ManagedMCGReconciler struct {
 	Log                          logr.Logger
 	Scheme                       *runtime.Scheme
 
-	ctx                         context.Context
-	images                      ImageMap
-	managedMCG                  *mcgv1alpha1.ManagedMCG
-	namespace                   string
-	noobaa                      *noobaav1alpha1.NooBaa
-	odfOperatorManagerConfigMap *v1.ConfigMap
-	operatorConsole             *operatorv1.Console
-	reconcileStrategy           mcgv1alpha1.ReconcileStrategy
-	storageCluster              *ocsv1.StorageCluster
-	storageSystem               *odfv1alpha1.StorageSystem
+	ctx               context.Context
+	images            ImageMap
+	managedMCG        *mcgv1alpha1.ManagedMCG
+	namespace         string
+	noobaa            *noobaav1alpha1.NooBaa
+	reconcileStrategy mcgv1alpha1.ReconcileStrategy
+
+	AddonParamSecretName string
+
+	prometheus                    *promv1.Prometheus
+	pagerdutySecret               *corev1.Secret
+	deadMansSnitchSecret          *corev1.Secret
+	smtpSecret                    *corev1.Secret
+	alertmanagerConfig            *promv1a1.AlertmanagerConfig
+	alertRelabelConfigSecret      *corev1.Secret
+	addonParams                   map[string]string
+	onboardingValidationKeySecret *corev1.Secret
+	alertmanager                  *promv1.Alertmanager
+	PagerdutySecretName           string
+	dmsRule                       *promv1.PrometheusRule
+	DeadMansSnitchSecretName      string
+	CustomerNotificationHTMLPath  string
+	SMTPSecretName                string
+	SOPEndpoint                   string
+	AlertSMTPFrom                 string
+	Route                         *routev1.Route
```

Comment:
Please separate the exported and non-exported fields with one newline.

Reply:
updated